src/epic-auth/epic-auth.module.ts

```diff
@@ -9,7 +9,7 @@ import { SmartOnFhirAuthService } from './smart-on-fhir-auth.service';
 let privateKey = ""
 
 try {
-    privateKey = fs.readFileSync('private_key.pem', 'utf8')
```

Comment:
same, revert all the changes to epic-auth module.. it will cause conflict.. i am deleting this already